src/components/Common/UI/DatePicker/DatePicker.tsx

```diff
@@ -25,25 +25,18 @@ import CaretDown from '@/assets/icons/caret-down.svg?react'
 import CaretRight from '@/assets/icons/caret-right.svg?react'
 import CaretLeft from '@/assets/icons/caret-left.svg?react'
 
-// Utility functions to convert between Date and DateValue
-function dateToCalendarDateValue(date: Date | null | undefined): DateValue | null {
-  if (!date) return null
-
-  return new CalendarDate(
-    date.getFullYear(),
-    date.getMonth() + 1, // JavaScript months are 0-indexed
-    date.getDate(),
-  )
-}
-
+// Function to convert DateValue back to Date
 function dateValueToDate(dateValue: DateValue | null): Date | null {
   if (!dateValue) return null
 
-  return new Date(
+  // Create new Date object in local timezone
```

Comment:
Should this be extracted to a function named `createNewLocalDate` or something like that instead of the comment?

Reply:
So this is just internal to DatePicker beacuse of how React-Aria handles dates in a non-standard way.

Reply:
Creating a function doesn't have to be to make it available outside the module!
https://refactoring.guru/smells/comments

Reply:
I would refactor this one out however DateValue is specific to React-Aria's dependency on their date library so this one i'm going to keep here so we can shake it away.